spec/vcr/unstub/ok.yml

```diff
@@ -0,0 +1,694 @@
+---
```

Comment:
Could we use any existing VCR cassette instead of this one? I'm also a bit worried about exposing this "real" data in this public repository.. May be we should filter out some data from recorded cassettes? what do you think?

Reply:
do you mean modifying the request body directly in the cassettes? about reusing a cassette. I can do it if I only test `search` or `profile` but not both for the case `makes real requests`

Reply:
Yes, we can edit response body directly in the cassette, but don't worry for now, we can do it later this week.
Lets just split `unstubbing the request after stubbing` context in two, so we can reuse existing cassettes